### ISSUE TITLE
Multi-line secret_environment_variables fails without a comma

### ISSUE THREAD

**Issue by mattdsteele** [google-github-actions/deploy-cloud-functions#308]
### TL;DR

As noted in #287, when trying to set multiple secrets as environment variables, I had to delimit them with a comma in order for the deploy to succeed.

### Expected behavior

Based on comments, I'd expect to not have to comma-delimit my environment variables, but I found it was required.

### Observed behavior

I received this error in my logs:

```
Error: google-github-actions/deploy-cloud-functions failed with: failed to parse secret reference "projects/***/secrets/STRAVA_CLIENT_ID
STRAVA_CLIENT_SECRET=projects/***/secrets/STRAVA_CLIENT_SECRET
STRAVA_ACCESS_TOKEN=projects/***/secrets/STRAVA_ACCESS_TOKEN
STRAVA_REFRESH_TOKEN=projects/***/secrets/STRAVA_REFRESH_TOKEN": unknown format. Secrets should be of the format "projects/p/secrets/s/versions/v".
```

### Action YAML

```yaml
name: Build and Deploy to Cloud Function

on:
  workflow_dispatch:
    branches: [main]
  push:
    paths:
      - backend/**
    branches:
      - main

env:
  PROJECT_ID: ${{ secrets.GCLOUD_PROJECT }}
  RUN_REGION: us-central1
  SERVICE_NAME: populate-trends
  ENTRY_POINT: GetLatestTrends
  TRIGGER_RESOURCE: projects/secret-strava/topics/tj-trends

jobs:
  setup-build-deploy:
    name: Setup, Build, and Deploy
    runs-on: ubuntu-latest

    steps:
    - name: Checkout
      uses: actions/checkout@v2

    - id: 'auth'
      uses: 'google-github-actions/auth@v0'
      with:
        credentials_json: '${{ secrets.GCLOUD_ACCOUNT_KEY }}'

    # Setup gcloud CLI
    - name: 'Set up Cloud SDK'
      uses: google-github-actions/setup-gcloud@v0
      with:
        project_id: ${{ secrets.GCLOUD_PROJECT }}
    # Deploy image to Cloud Run
    - name: 'deploy function'
      uses: 'google-github-actions/deploy-cloud-functions@v0'
      with:
        name: "${{ env.SERVICE_NAME }}"
        runtime: 'go116'
        entry_point: "${{ env.ENTRY_POINT }}"
        secret_environment_variables: |-
          STRAVA_CLIENT_ID=projects/secret-strava/secrets/STRAVA_CLIENT_ID
          STRAVA_CLIENT_SECRET=projects/secret-strava/secrets/STRAVA_CLIENT_SECRET
          STRAVA_ACCESS_TOKEN=projects/secret-strava/secrets/STRAVA_ACCESS_TOKEN
          STRAVA_REFRESH_TOKEN=projects/secret-strava/secrets/STRAVA_REFRESH_TOKEN
        event_trigger_type: 'google.pubsub.topic.publish'
        event_trigger_resource: "${{ env.TRIGGER_RESOURCE }}"
        source_dir: 'backend'
```

### Log output

```text
Failed run: https://github.com/mattdsteele/strava-segment-trends/runs/5606007064?check_suite_focus=true

Successful run: https://github.com/mattdsteele/strava-segment-trends/actions/runs/2006156479
```

### Additional information

For what it's worth, I don't think it's unreasonable or unexpected behavior to require a comma, so I'm mostly posting this for completeness (or as an update to the documentation). Thanks!
